dogpile/cache/backends/file.py

```diff
@@ -140,6 +140,13 @@ def release_write_lock(self):
     """
 
     def __init__(self, arguments):
+        super().__init__(
+            {
+                "pickler": pickle.dumps,
```

Comment:
can I send my own pickler here?    seems like not?

Reply:
yes, the idea here is that if "pickle" is not in arguments, then we default it to pickle.dumps:
```python
>>> arguments = {}  # No arguments from user
>>> {
...     "pickler": pickle.dumps,
...     "unpickler": pickle.loads,
...     **arguments
... }
{'pickler': <built-in function dumps>, 'unpickler': <built-in function loads>}

>>> def noop(x): return x
...
>>> arguments = {"pickler": noop, "unpickler": noop}  # User overrides
>>> {
...     "pickler": pickle.dumps,
...     "unpickler": pickle.loads,
...     **arguments
... }
{'pickler': <function noop at 0x102b7a0d0>, 'unpickler': <function noop at 0x102b7a0d0>}
```